lib/utils/ec_key.ex

```diff
@@ -44,6 +44,15 @@ defmodule ArkEcosystem.Crypto.Utils.EcKey do
     |> private_key_to_address(network_address)
   end
 
+  def pubkey_from_hex(public_key) do
+    BtcCore.decompress(public_key)
+  end
+
+  def verify_message(message, signature, public_key) do
```

Comment:
This method should be inside `message.ex` as it is specific to the message entity.